task/python-boto3/0.1/python-boto3.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: python-boto3
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: CLI
+    tekton.dev/tags: CLI, boto3, sdk
+    tekton.dev/displayName: "python boto3 aws"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: >-
+    Executes a Python script using AWS Boto3 to manage and inspect AWS resources.
+  params:
+    - name: aws-region
+      type: string
+      default: "us-east-1"
+      description: AWS Region
+  volumes:
+    - name: python-script
+      configMap:
+        name: python-script-configmap
+  steps:
+    - name: run-python-script
+      image: python:3.9
+      env:
+        - name: AWS_ACCESS_KEY_ID
+          valueFrom:
+            secretKeyRef:
+              name: aws-credentials
+              key: access-key-id
+        - name: AWS_SECRET_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              name: aws-credentials
+              key: secret-access-key
```

Comment:
can we document this in readme properly? users using this task might miss out these details. Alternatively, can we parameterize the secret name and the keys so that it gives more control to the user?

Reply:
done